pkg/db/migration/convert/convert000031/convert.go

```diff
@@ -0,0 +1,104 @@
+package convert000031
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+
+	"github.com/instill-ai/pipeline-backend/pkg/datamodel"
+)
+
+const batchSize = 100
+
+type ConvertToWeb struct {
+	DB     *gorm.DB
+	Logger *zap.Logger
+}
+
+func (c *ConvertToWeb) Migrate() error {
+	log.Println("=== Start migration ===")
+	if err := c.migratePipeline(); err != nil {
+		return err
+	}
+
+	return c.migratePipelineRelease()
+}
+
+func (c *ConvertToWeb) migratePipeline() error {
+	pipelines := make([]*datamodel.Pipeline, 0, batchSize)
+	return c.DB.Select("uid", "recipe_yaml", "recipe").FindInBatches(&pipelines, batchSize, func(tx *gorm.DB, _ int) error {
+		for _, p := range pipelines {
+			l := c.Logger.With(zap.String("pipelineUID", p.UID.String()))
+
+			recipeYAML := p.RecipeYAML
+
+			updatedRecipeYAML, isUpdate := c.updateRecipeYAML(recipeYAML)
+
+			log.Println("Updated Recipe YAML: \n", updatedRecipeYAML)
+
+			log.Println("Is Update: ", isUpdate)
```

Comment:
🙏 please remove the logs if not necessary. Also, you've initialised `zap.Logger` as `l` but then you're using a different package to log.